Dockerfile-ubuntu

```diff
@@ -32,11 +36,6 @@ RUN useradd --create-home --shell /bin/bash pdagent \
   # Primarily for handling sudo use during build and testing.
   && echo "pdagent ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers
 
-RUN apt-get update \
-    && apt-get install -y systemd \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
-
```

Comment:
Moved these up to take better advantage of step caching.

Reply:
Ahh good call